### ISSUE TITLE
Shake Hands session

### ISSUE THREAD

**Issue by bnomei** [getkirby/konf#9]
could be done right at the start of konf. 

everyone participating in the session lines up. other walk past them. hands are shaken and names/aliases named.

pros: looks funny. 

cons: great for spreading a spring flu.

**Comment by dervondenbergen** [comment]
Something I saw some days ago at a barcamp/unconference was that everyone had to describe themselves at the start with three hashtags. This could also spark some interesting (non-kirby) conversations.

**Comment by lukasbestle** [comment]
To be honest, both ideas sound a bit constrained to me, not as casual as the community usually is.

**Comment by bnomei** [comment]
@lukasbestle personally i would enjoy the few minutes to really see everyone face to face and know their name. specially for those "weirdos" that do not want they avatar picture to be a photo – it might be nice to get to know their face. like "ah yes this is @lukasbestle, he does not look like engine wheels at all." ;-)

i mainly suggested it since it might speed up getting to know each other and time is limited at a konf. but hey... its just a suggestion so if enough ppl think its not needed or we have enough other session then we just dont do it.

**Comment by lukasbestle** [comment]
I fully agree on the idea to have some kind of introduction session with everyone. After all it's a community meetup, not a formal conference. But maybe we'll find something that's more compatible with people who don't want to be in the spotlight.

> ah yes this is @lukasbestle, he does not look like engine wheels at all

No no, I do actually look like engine wheels. You'll see. :P

**Comment by bnomei** [comment]
i will change the issue name to refect the `introduction session` idea

**Comment by johannahoerrmann** [comment]
I actually like these ideas, minus spreading the flu. In addition / instead of exclaiming one's username or hashtags, everyone could write them out as a name tag (could be something fancy on a lanyard or a simple piece of painter's tape).
